### PR TITLE
Fixes.

### PR DESCRIPTION
**Pull request by awholegunch**
Minor doc fixes to get a clean build.

Passes [Quick Build 2](http://builds.cask.co/browse/CDAP-DQB90-2)

### CODE REVIEW

**Comment by bdmogal** [comment]
lgtm

File: cdap-docs/hydrator-manual/build.sh [hunk 1]
```diff
@@ -233,7 +233,7 @@ function download_includes() {
   download_md_file core-plugins File-batchsource.md
   download_md_file core-plugins FTP-batchsource.md
   download_md_file core-plugins GroupByAggregate-batchaggregator.md
-  download_md_file core-plugins HDFSFileMoveAction-action.md
+  download_md_file core-plugins HDFSMove-action.md
```
Comment (sreevatsanraman) [inline cdap-docs/hydrator-manual/build.sh:236]: Missing atlest HDFSDelete-action and Database-action. Let's track this if we can't fix now.

**Comment by sreevatsanraman** [comment]
LGTM.

File: cdap-docs/hydrator-manual/build.sh (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/hydrator-manual/build.sh:236]: That's in the other PR. This is just to get a clean build in the main release line.

**Pull request merged**